modules/swagger-codegen/src/main/resources/typescript-node/package.mustache

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "{{npmName}}",
+    "version": "{{npmVersion}}",
+    "description": "NodeJS client for {{npmName}}",
+    "main": "api.js",
+    "scripts": {
+        "build": "typings install && tsc"
+    },
+    "author": "Mads M. Tandrup",
```

Comment:
Please change it to "Swagger Codegen Contributors" and I'll give a heads-up to Mads on this.